workers/unity/Packages/io.improbable.gdk.tools/.CodeGenTemplate/CodeGenerationLib/Model/Details/DetailsStore.cs

```diff
@@ -4,37 +4,40 @@
 using System.Collections.ObjectModel;
 using System.Linq;
 using Improbable.Gdk.CodeGeneration.FileHandling;
+using Improbable.Gdk.CodeGeneration.Utils;
 using NLog;
 
 namespace Improbable.Gdk.CodeGeneration.Model.Details
 {
     public class DetailsStore
     {
-        public IReadOnlyDictionary<string, UnityTypeDetails> Types { get; }
-        public IReadOnlyDictionary<string, UnityEnumDetails> Enums { get; }
-        public IReadOnlyDictionary<string, UnityComponentDetails> Components { get; }
-        public readonly ImmutableHashSet<string> BlittableSet;
-        public IReadOnlyList<string> SchemaFiles { get; }
+        public readonly IReadOnlyDictionary<string, UnityTypeDetails> Types;
+        public readonly IReadOnlyDictionary<string, UnityEnumDetails> Enums;
+        public readonly IReadOnlyDictionary<string, UnityComponentDetails> Components;
 
-        public static readonly HashSet<PrimitiveType> NonBlittableSchemaTypes = new HashSet<PrimitiveType>
-            { PrimitiveType.Bytes, PrimitiveType.String, PrimitiveType.Entity };
+        public readonly ImmutableHashSet<string> BlittableSet;
+        public readonly IReadOnlyList<string> SchemaFiles;
 
-        public IFileTree FileTree { get; }
+        private static readonly HashSet<PrimitiveType> NonBlittableSchemaTypes = new HashSet<PrimitiveType>
+        {
+            PrimitiveType.Bytes,
+            PrimitiveType.String,
+            PrimitiveType.Entity
+        };
```

Comment:
Would it be cleaner to add an extension method on `PrimitiveType`: 

```
bool IsBlittable()
```

which returns ```this != PrimitiveType.Bytes && this != ...```

Reply:
yes - it is a lot cleaner now